### PR TITLE
JRuby support

### PR DESCRIPTION
**Pull request by dometto**

### CODE REVIEW

File: test/test_wiki.rb [hunk 1]
```diff
@@ -539,7 +539,6 @@
 context "page_file_dir option" do
   setup do
     @path          = cloned_testpath('examples/page_file_dir')
-    @repo          = Gollum::Git::Repo.init(@path)
```
Comment (dometto) [inline test/test_wiki.rb]: `@repo` not being used in this context, so this line is unnecessary.

**Comment by dometto** [comment]
\0/

@bartkamphorst, @sunny: what say you?

Would it be a good idea to move https://github.com/repotag/gollum-lib_rjgit_adapter to the gollum org (and add travis tests, coverage, etc.)?

File: test/test_wiki.rb (hunk 1 above)
Comment (bartkamphorst) [inline test/test_wiki.rb]: Fair enough.

**Comment by bartkamphorst** [comment]
Exciting to see all the lights green! :tada: Nice work with the gemspec. I'm in favor of merging.

Adding Travis, coverage, etc. is also a good idea at this point, moving the gem can be decided at a later time I think.

**Comment by sunny** [comment]
Good job there, also like the way the gemspec is gemerated!

Shouldn't `lib/version.rb` be under `lib/gollum-lib/`?

File: gemspec.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
+def specification(version, default_adapter, platform = nil)
+  Proc.new do |s|
+    s.specification_version = 2 if s.respond_to? :specification_version=
+    s.required_rubygems_version = Gem::Requirement.new('>= 0') if s.respond_to? :required_rubygems_version=
+    s.rubygems_version = '0.0.1'
```
Comment (bartkamphorst) [inline gemspec.rb]: Should this really be 0.0.1?

**Comment by dometto** [comment]
Good point @sunny :) Are you also ok with using kramdown als the default markdown gem for gollum ( https://github.com/gollum/gollum/pull/994)? I was thinking of bumping gollum to 4.0 after that.

**Comment by sunny** [comment]
Yup, sounds good to me.

**Pull request merged**